python/paddle/fluid/contrib/slim/quantization/quantization_mkldnn_pass.py

```diff
@@ -375,11 +375,15 @@ def _remove_fake_ops(self, graph):
             if op.name() in self._fake_quantize_types:
                 op_out = graph._find_node_by_name(op.outputs,
                                                   op.output("Out")[0])
-                self._remove_fake_quantize(graph, op)
+                next_op = op_out.outputs[0]
+                if next_op.name() not in self._mul_ops:
+                    self._remove_fake_quantize(graph, op)
```

Comment:
What is this ` if next_op.name() not in self._mul_ops` for?

Reply:
Because we will transform the fake_quantize op before mul to quantize op.